Config.psm1

```diff
@@ -21,57 +21,74 @@ Import-Module "$localResourcesDir\ini.psm1"
-          "Description" = ""},
+          "Description" = "This parameter can be set to either BIOS or UEFI."},
         @{"Name" = "product_key";
-          "Description" = ""},
+          "Description" = "The product key for the selected OS."},
         @{"Name" = "extra_features";
-          "Description" = ""},
+          "Description" = "An comma separated array of extra features that will be enabled on the resulting image.
+                           These features need to be present in the ISO file."},
         @{"Name" = "force"; "DefaultValue" = $false; "AsBoolean" = $true;
-          "Description" = ""},
+          "Description" = "It will force the image generation when $RunSysprep is $False or the selected $SwitchName
+                           is not an external one. Use this parameter with caution because it can easily generate
+                           unstable images."},
         @{"Name" = "install_maas_hooks"; "DefaultValue" = $false; "AsBoolean" = $true;
-          "Description" = ""},
+          "Description" = "If set to true, MaaSHooks will be installed."},
         @{"Name" = "zip_password";
-          "Description" = ""},
+          "Description" = "If this parameter is set, after the image is generated, 
+                           a password protected zip archive with the image will be created."},
         @{"Name" = "administrator_password"; "GroupName" = "vm"; "DefaultValue" = "Pa`$`$w0rd";
-          "Description" = ""},
+          "Description" = "This will be the administrator's password before the image starts the sysprep process."},
         @{"Name" = "external_switch"; "GroupName" = "vm"; "DefaultValue" = "external";
-          "Description" = ""},
+          "Description" = "Used to specify the virtual switch the VM will be using to connect to the internet.
+                           If none is specified, one named external will be created. 
+                           If it is specified but not present, it will also be created. "},
         @{"Name" = "cpu_count"; "GroupName" = "vm"; "DefaultValue" = "1";
-          "Description" = ""},
+          "Description" = "The number of CPU cores assigned to the VM used to generate the image."},
         @{"Name" = "ram_size"; "GroupName" = "vm"; "DefaultValue" = "2147483648";
-          "Description" = ""},
+          "Description" = "RAM(in bytes) assigned to the VM used to generate the image."},
         @{"Name" = "disk_size"; "GroupName" = "vm"; "DefaultValue" = "42949672960";
-          "Description" = ""},
+          "Description" = "Disk space (in bytes) assigned to the VM used to generate the image."},
         @{"Name" = "virtio_iso_path"; "GroupName" = "drivers";
-          "Description" = ""},
+          "Description" = "The path to the ISO file containing the VirtIO drivers."},
         @{"Name" = "virtio_base_path"; "GroupName" = "drivers";
-          "Description" = ""},
+          "Description" = "The location where the VirtIO drivers are found."},
```

Comment:
For example, the location of a mounted VirtIO ISO. VirtIO versions supported >=0.1.6.x